api-reference/beta/resources/permissionscope.md

```diff
@@ -0,0 +1,56 @@
+# permissionScope resource type
```

Comment:
See https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/entity-and-complex-type-reference#oauth2permission-type for the intro and descriptions